src/main/java/com/github/kokorin/jaffree/util/ParseUtil.java

```diff
@@ -104,7 +105,10 @@ public static Long parseSizeInKiloBytes(final String value) {
             return null;
         }
 
-        return parseLongWithSuffix(value.trim(), KBYTES_SUFFIX);
+        final String trimmedValue = value.trim();
+        final Matcher matcher = KBYTES_SUFFIX_PATTERN.matcher(trimmedValue);
+
+        return matcher.find() ? parseLongWithSuffix(trimmedValue, matcher.group(1)) : null;
```

Comment:
I believe regex is an overkill here,: there are only 2 possible suffixes. So it could be fine to check first for `KiB` and then for `kB`. 
Besides that, I checked ffmpeg source code and it prints **kibi** bytes even with `kB` suffix. So I think it makes sense to set multiplier to `1024` instead of `1000` in `parseSizeInBytes` function.

Please, consider adding unit test to parse `KiB` suffix.